content/blog/2015-08-xx-service-discovery-etc.md

```diff
@@ -0,0 +1,318 @@
+---
+title: Custom service discovery with etcd
+created_at: 2015-06-23
+kind: article
+author_name: Fabian Reinartz 
+---
+
+In a [previous post](/blog/2015-06-01-advanced-service-discovery.md) we
+introduced numerous new ways of doing service discovery in Prometheus.
+Since then a lot has happened. We improved the internal implementation and
+received fantastic contributions from our community, adding support for
+service discovery with Kubernetes and Marathon. They will become available
+with the release of version 0.16.
+
+We also touched the topic of [custom service discovery](blog/2015-06-01-advanced-service-discovery.md#custom-service-discovery).
```

Comment:
Oh, due to only reading your comment "More common == better" in an email thread, I thought it was related to the "based on" comment. In this case here, I'd really favor "touched on" because that's the phrase that really has the meaning you are going after here ("To mention briefly, or in passing."), while just touching a topic is not something one would usually say in this way.